frontend/src/views/pricing-view.js

```diff
@@ -0,0 +1,151 @@
+import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
+import '../components/ex-card.js';
+
+/**
+ * `pricing-view`
+ *
+ * PricingView element.
+ *
+ * @customElement
+ * @polymer
+ */
+class PricingView extends PolymerElement {
+
+    static get template() {
+        return html`
+            <style include="common-styles">
+                :host {
+                    display: block;
+                }
+                
+                .wrapper {
+                    margin: 0 auto; /* Horizontally center the layout grid if there is space around it */
+                    max-width: 960px;
+                    padding: 0 var(--lumo-space-m);
+                }
+
+                #header {
+                    padding: var(--lumo-space-wide-l);
+                    margin-bottom: var(--lumo-space-xl);
+                    box-shadow: 0 0 0 1px var(--lumo-contrast-5pct), var(--lumo-box-shadow-xs);
+                }
+                
+                #header h5 {
```

Comment:
Out of curiosity, why is the heading level for company name so low (`h5`) ?

This view doesn't seem to need more than 3 levels of headers (taking into account footer columns etc).
So let's use `h2` or `h3` and then style them as needed to get proper visual appearance.

Reply:
This layout is based on https://getbootstrap.com/docs/4.3/examples/pricing/ so some of the heading levels were just copied from there.

Actually I think the company name at the top should not even be an `<h*>` element, but something else since it's not really a heading for any section.

Mainly the important thing is that the main page title "Pricing" is `<h1>` which it is. Othwerwise afaik the header levels don't really matter as long as they are in a logical order in the DOM for having proper nesting order/hierarchy. Afaik skipping levels doesn't matter so the numbers don't need to be strictly sequential.

I think semantically it's ok as it is except maybe I'd like to change the company name to a `<div>` instead. But I'm ok with changing the footer headings to `<h2>` if you think that's better?

Reply:
Or maybe company name can also be `<h2>`. At least I don't see a real need for any more than two heading levels here.

Reply:
I think `h2` would make more sense. There are different views on whether company name should be `h1` or not, and I don't have strong opinion here.